test/testdata/golden/gapic/lang/csharp_gapic.yaml

```diff
@@ -0,0 +1,5 @@
+type: com.google.api.codegen.ConfigProto
```

Comment:
Originally, I compare the whole output content. Having a static set of input can avoid unnecessary noise caused by the change to googleapis repo itself. Assuming submodule won't update by itself, I can switch to submodule googleapis repo, except that there is no gapic file for library-example at this moment. Anyway to add something into a submodule directory?